projects/packages/stats/src/class-wpcom-stats.php

```diff
@@ -0,0 +1,219 @@
+<?php
+/**
+ * Stats WPCOM_Stats
+ *
+ * @package automattic/jetpack-stats
+ */
+
+namespace Automattic\Jetpack\Stats;
+
+use Automattic\Jetpack\Connection\Client;
+use Jetpack_Options;
+
+/**
+ * Stats WPCOM_Stats class.
+ *
+ * Responsible for fetching Stats related data from WPCOM.
+ *
+ * @since $$next-version$$
+ */
+class WPCOM_Stats {
+	/**
+	 * WPCOM Stats CSV Url.
+	 */
+	const STATS_CSV_URL = 'https://stats.wordpress.com/csv.php';
+
+	/**
+	 * Get stats from WordPress.com
+	 *
+	 * @param string $table The stats which you want to retrieve: postviews, or searchterms.
+	 * @param array  $args {
+	 *      An associative array of arguments.
+	 *
+	 *      @type bool    $end        The last day of the desired time frame. Format is 'Y-m-d' (e.g. 2007-05-01)
+	 *                                and default timezone is UTC date. Default value is Now.
+	 *      @type string  $days       The length of the desired time frame. Default is 30. Maximum 90 days.
+	 *      @type int     $limit      The maximum number of records to return. Default is 10. Maximum 100.
+	 *      @type int     $post_id    The ID of the post to retrieve stats data for
+	 *      @type string  $summarize  If present, summarizes all matching records. Default Null.
+	 *
+	 * }
+	 *
+	 * @return array {
+	 *      An array of post view data, each post as an array
+	 *
+	 *      array {
+	 *          The post view data for a single post
+	 *
+	 *          @type string  $post_id         The ID of the post
+	 *          @type string  $post_title      The title of the post
+	 *          @type string  $post_permalink  The permalink for the post
+	 *          @type string  $views           The number of views for the post within the $num_days specified
+	 *      }
+	 * }
+	 */
+	public static function get_csv( $table, $args = null ) {
```

Comment:
Minor: This could be refactored to a class of its own to avoid the generic `$args` that we can only describe via comments right now.